src/SignalR/clients/ts/FunctionalTests/ts/HubConnectionTests.ts

```diff
@@ -49,12 +49,12 @@ function getConnectionBuilder(transportType?: HttpTransportType, url?: string, o
 }
 
 describe("hubConnection", () => {
-    eachTransportAndProtocol((transportType, protocol) => {
+    eachTransportAndProtocolAndHttpClient((transportType, protocol, httpClient) => {
```

Comment:
How many new test cases does this add? How much longer does this make the tests run?

Reply:
~100 tests, took ~13 seconds

Reply:
I'll allow it 😉. We can always do a cleanup later where we remove extranious variations.